src/lib/web-component/filter-timestamp/filter-timestamp-element.js

```diff
@@ -33,14 +35,15 @@ export class FilterTimestamp extends HTMLElement {
 
     const timeStamp = this.timeStamp || {};
 
-    const chosenTimestamp = selectedRow.querySelector('.timestamp').textContent;
+    const chosenTimestamp = selectedRow.querySelector('.timestamp')?._timestamp;
```

Comment:
Use `selectedRow._log.timeStamp` instead.

I would rather not have an external class using an internal `_log` property, but it is preferable to your current approach, of adding a `_timeStamp` property to an unrelated DOM element.

To reduce the risk of future regressions, it would be nice to have a unit test that verifies that the time labels are rendered as expected.